pkg/dohdec-cli/lib/cli.js

```diff
@@ -86,6 +86,7 @@ export class DnsCli extends Command {
       )
       .option('-d, --dns', 'Use DNS format instead of JSON (ignored for TLS)')
       .option('-s, --dnssec', 'Request DNSsec records')
+      .option('-k, --dnssecCheckingDisabled', 'Disable DNSsec validation')
```

Comment:
I'm not super happy with the `-k` (taken from `chec_k_ing`) but couldn't find another relevant letter that wasn't already taken.